docs/bpftool-gen.rst

```diff
@@ -31,182 +31,164 @@ GEN COMMANDS
+        used to set up initial values of variables, if set before 
+        **example__load**. Afterwards, if target kernel supports memory-mapped 
+        BPF arrays, same structs can be used to fetch and update (non-read-only) 
+        data from userspace, with same simplicity as for BPF side.
+
+    bpftool gen subskeleton *FILE*
+        Generate BPF subskeleton C header file for a given *FILE*.
+
+        Subskeletons are similar to skeletons, except they do not own the 
+        corresponding maps, programs, or global variables. They require that 
+        the object file used to generate them is already loaded into a 
+        *bpf_object* by some other means.
+
+        This functionality is useful when a library is included into a larger 
+        BPF program. A subskeleton for the library would have access to all 
+        objects and globals defined in it, without having to know about the 
+        larger program.
+
+        Consequently, there are only two functions defined for subskeletons:
+
+            example__open(bpf_object\*)
+            Instantiates a subskeleton from an already opened (but not 
+            necessarily loaded) **bpf_object**.
+
+            example__destroy()
+            Frees the storage for the subskeleton but *does not* unload any BPF 
+            programs or maps.
+
+    bpftool gen min_core_btf *INPUT* *OUTPUT* *OBJECT* [*OBJECT*...] 
+        Generate a minimum BTF file as *OUTPUT*, derived from a given *INPUT* BTF file, containing all needed BTF types so one, or more, given eBPF objects CO-RE relocations may be satisfied.
+
+        When kernels aren't compiled with CONFIG_DEBUG_INFO_BTF, libbpf, when 
+        loading an eBPF object, has to rely on external BTF files to be able to 
+        calculate CO-RE relocations.
+
+        Usually, an external BTF file is built from existing kernel DWARF data 
+        using pahole. It contains all the types used by its respective kernel 
+        image and, because of that, is big.
+
+        The min_core_btf feature builds smaller BTF files, customized to one 
+        or multiple eBPF objects, so they can be distributed together with an 
+        eBPF CO-RE based application, turning the application portable to 
+        different kernel versions.
+
+        Check examples bellow for more information how to use it.
+
+    bpftool gen help
+        Print short help message.
 
 OPTIONS
 =======
 	.. include:: common_options.rst
 
 	-L, --use-loader
-		  For skeletons, generate a "light" skeleton (also known as "loader"
-		  skeleton). A light skeleton contains a loader eBPF program. It does
-		  not use the majority of the libbpf infrastructure, and does not need
-		  libelf.
+		For skeletons, generate a "light" skeleton (also known as "loader"
+		skeleton). A light skeleton contains a loader eBPF program. It does
+		not use the majority of the libbpf infrastructure, and does not need
+		libelf.
```

Comment:
These (and `include` above) are still using tabs.

Reply:
Corrected.